### PR TITLE
Update Tor safety instructions on Source Interface per Tor Browser UI changes

### LINKED ISSUE #4465: Update purple Tor Safest bar on Source Interface per Tor Browser new UX & urgent usability needs
## Description

The Tor team eliminated the Security Slider in it's latest Browser release, moving that functionality into the browser's Preferences pane. A huge win for general-population usability (even though power-users are likely to dread this)! 😸 

As such:
1. The iconography in the toolbar users are directed to, has changed.
2. The iconography and text in the flyout are now obsolete.
3. Because the experience has changed, the directives in the flyout need to change.

In a parallel effort, updates to the Source experience around motivating users to comply with Tor stuff, has also been getting explored in #4334. As a part of that effort, a recommendation was made to make asap improvements to this piece of the UI (the purple bar) within today's experience as an asap precursor to deeper experiential improvements. 
- Today's copy is abundant with "filler" text to express complete, fully accurate thoughts in a polite and helpful fashion. To users this unfortunately gets conflated with marketing content in split-second, unconscious decisions to tune-out what our modern brains have been trained to believe is most likely unimportant fluff. Which sucks, because this is neither.  
  - Tightly structuring messaging content in brief chunks that are rarely complete sentences, is a known UX best-practice for copywriting for driving user behavior. The "Copywriting for UI" section I put together in the [Resources block on the SD UX Wiki ](https://github.com/freedomofpress/securedrop-ux/wiki#resources) lists some great resources to learn more about this. 
- A simple "bang" icon at the beginning of a shortened sentence is likely to grab the user's attention, more than simply starting it with words; and reducing the sentence's footprint entices engagement. 
  - It's also become common for companies and webapps to announce new features or things they want you to buy, in full-width micro-banner messaging like this. This message is not that, and the little bang can go a long way communicating this quickly.

This ticket merges the asap recommendation from #4334, with the immediate needs cited in #4454 to also update messaging per the latest Tor Browser updates. 

## User Research Evidence

This section should be renamed to: _**User Needs This Issue Addresses**_. Labeling this section a place to cite evidence of research that rarely exists dilutes the value using _actual_ user research evidence. 

That said, user needs are adequately laid forth in the above cross-referenced tickets. 

## Implementation Guidelines

I've prioritized the updates in bullets below the mockup. 
MUST, SHOULD, and COULD. 
Out of respect for dev time, I'll only insist that MUSTs get implemented—but SHOULD items also fit within my criteria for addressing urgent usability needs in a high-impact/low-effort fashion, and if at all possible I'd like them to get implemented, too. The one COULD task may be a huge PITA, and is less important than the SHOULDs.

image

- Update banner text, to: `Your Tor Browser’s Security Setting is too low.  Use the       button in your browser’s toolbar to change it.`
  - MUST
- Hyperlink and bold `Security Setting` and present as a proper noun (m/c) as the active cue in the text
  - MUST
     - Hyperlinked text should always be of the _destination_, not of a concept you're asking the user to buy into. Yep, "Learn More" falls into the latter as a best-practice; it just feels too optional, and queue's a decisionmaking process vs stating plainly a place we're directing the user to go to.
- Include [_Guard_ icon](https://github.com/freedomofpress/securedrop/issues/4465#issuecomment-495943892) (what Torproject is calling it) and [_Bang_ icon](https://github.com/freedomofpress/securedrop-ux/issues/17#issuecomment-472172628) in the bar
  - SHOULD
     - This icon is referenced as the Shield in other tickets; but Tor calls it the Guard icon, and I recommend consistent verbiage to make everyone's life easier. Even though I agree, it's a friggin' shield.
     - Visuals do much more to drive user action (when used correctly), than words. 
     - This is not a happy/cheery "Yay, new features we want to tell you about!" message. The user needs a symbolic cue to make the case for them to read the next 18 words. 
- Remove "x" to collapse the bar
  - SHOULD
    - Easy to do; it's against the user's best interest to enable their blowing this off
- Update bubble text (content in code block, below): 
  - MUST
      - The slider is dead. Its functionality has been moved to a set of radio buttons in the browser's _preferences_ pane, and as such today's existing text is misleading. There is no longer a _slider_ control for a new user to find.
      - Eliminate all text not included in code block, above. The intro pleasantries are well intended, but unfortunately distract from task completion. 
      - In item number one, the shield icon should be placed in the big space between "the" and "in"
      - Hyperlinked text should be "Refresh this page" per rationale cited above for linked text in purple bar 
```
1. Click the        in your browser’s toolbar above
2. Select Advanced Security Settings
3. Select Safest

Refresh this page, and you’re done!
```
- Update the current Guard icon to the cleaner SVG (code in the comment of this Issue)
  - MUST
    - Guard icon in the PR's code is obfuscated by unnecessary negative space in a square graphic. Icons or icons w/in instructional content are only valuable if users can quickly and easily 1:1 match the image in instructions, to a UI element. Too much _parsing_ is required of the image currently used.
- Eliminate the "x" in the flyout bubble
  - SHOULD
    - Easy to do, and it's in the user's best interest for us to force them to just make the damn change
- Tweak the pointer-thingy in the white bubble, so that it more simply points "up" vs specifically to a thing on the right or the left. 
   - COULD
      - The pointer's current styling infers more specificity than can realistically be built-in, and making the triangle of the pointer a horizontally-symmetrical triangle is a recommended improvement.

SVGs are pasted below in comments.

### PR DESCRIPTION
**Pull request by drewmassey**
## Status

Ready for Review

## Description of Changes

Fixes #4465 .

Changes proposed in this pull request:
- Deals with some consequences of design changes to Tor 8.5, as outlined by @ninavizz.

## Testing

How should the reviewer test this PR?
- Confirm that the local app and docs are up to date

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make -C securedrop test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

### If you made changes to documentation:

- [ ] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

**Comment by zenmonkeykstop** [comment]
The functional tests run against Tor Browser 8.0.x, which might be a factor. I'll try running them locally and report back.

**Comment by rmol** [comment]
I ran them locally earlier without error. :man_shrugging: 

But I should probably update the Dockerfiles to 8.5 next week.

**Comment by drewmassey** [comment]
This is giving a “no such container” issue in circleci but otherwise seems to pass. Is this a known testing issue, such as a race condition? I don’t have permissions to rerun the workflow.

**Comment by zenmonkeykstop** [comment]
Passed locally for me as well, gave the failing test a kick.

**Comment by drewmassey** [comment]
@zenmonkeykstop It looks like your run passed; I wonder if the issue is that the `run tests` step in circleCI should stop gracefully if it doesn't locate the container? It looks like it complains about not finding the container and then tries to run it anyway, resulting in that step of the pipeline hanging after 10 minutes with no output.

https://circleci.com/gh/freedomofpress/securedrop/28864#tests/containers/1

I haven't fully wrapped my mind around the securedrop CircleCI setup just yet  but I have seen this issue of containers not reliably spinning up in circleCI before, resulting in spooky intermittent test failures. My first guess is that I don't exactly understand why the docker container is explicitly deleted here:

https://github.com/freedomofpress/securedrop/blob/develop/.circleci/config.yml#L159

**Comment by redshiftzero** [comment]
the build failure you're talking about looks like it's CI issue #4480 (if you have an idea for resolution worth doing in a separate PR)

**Comment by drewmassey** [comment]
Yeah I saw that other issue, my guess is the `docker rm` command I mention above might be fishy, git blame says @redshiftzero put it in so I don’t know if you have a reason for it I’m not thinking of. 

Can we resolve this present PR since this issue is documented elsewhere and I will take a deeper dive on #4480 after I tackle #4280 as I promised Nina yesterday? I try to work on exactly one issue at a time when I’m an OSS interloper so as not to demand too much on the maintainers.

**Comment by ninavizz** [comment]
@drewmassey That issue was just collecting dust; I trust it to get resolved far more quickly as you're able to, than it would otherwise—really, no rush! :) It's a far bigger/broader thing the team has been working on for months/years, getting the infrastructure and dev workflow in a good place. I suspect a set of fresh eyes and a fresh brain, could do a lot of good to that cause, too.

File: securedrop/source_templates/index.html [hunk 1]
```diff
@@ -88,15 +88,14 @@ <h2 class="welcome-text">
...
-    <div class="bubble" id="security-slider-info" class="info">
-        <div class="row-reverse"> <img id="security-slider-info-close" class="close" src="{{ url_for('static', filename='i/font-awesome/black/times.svg') }}" width="20px" height="20px"></div>
-      <p>{{ gettext('You appear to be using the Tor Browser. You can move the Security Slider to Safest in four steps.') }}</p>
+    <div class="bubble" id="security-setting-info" class="info">
       <ol>
-        <li>{{ gettext('Click the <img src="{icon}" alt="shield icon"> shield icon in the toolbar above').format(icon=url_for("static", filename="i/shield.png"))  }}</li>
-        <li>{{ gettext('Click <strong>Advanced Security Settings...</strong>') }}</li>
-        <li>{{ gettext('Click <strong>Safest</strong>') }}</li>
-        <li>{{ gettext('<a href="/">Click here</a> to refresh the page') }}</li>
+        <li>{{ gettext('Click the <img src="{icon}" alt="shield icon"> in the toolbar above').format(icon=url_for("static", filename="i/font-awesome/black/guard.svg"))  }}</li>
```
Comment (zenmonkeykstop) [inline securedrop/source_templates/index.html]: Shield icon is incorrect, should be outlined shield, not solid. It changes to solid once `Safest` is selected.

File: docs/source.rst [hunk 2]
```diff
@@ -52,20 +52,19 @@ organization you are submitting to.
 |Source Interface with Javascript Disabled|
 
 If this is the first time you're using the Tor browser, it's likely that you
-have JavaScript enabled and that the Security Slider that Tor browser provides
+have JavaScript enabled and that the Security Setting that Tor browser provides
```
Comment (zenmonkeykstop) [inline docs/source.rst]: preexisting nit: "the Tor browser"

File: securedrop/source_templates/index.html (hunk 1 above)
Comment (ninavizz) [reply securedrop/source_templates/index.html]: @zenmonkeykstop We're advising the user to perform the action when the browser is _not_ in `safest`, though—does the whole banner not go away once the browser is at `safest`? It should.

Comment (zenmonkeykstop) [reply securedrop/source_templates/index.html]: Yes, that's what I'm getting at. When the browser is in `Standard` mode, the shield icon in the toolbar is just an outline. The instructions in the popup ask the user to click on the solid shield icon, which isn't displayed at that point.

Comment (ninavizz) [reply securedrop/source_templates/index.html]: Thank you for the clarification and for catching that! Ya, I couldn't find any specs (or even the artwork itself) for its implementation anywhere, so just kinda guessed. Submitted a new comment in the OG issue #4465 with updated SVG code.

File: docs/source.rst (hunk 2 above)
Comment (drewmassey) [reply docs/source.rst]: I will see your `nit` and raise you consistent capitalization throughout the doc :-) (eabb86ab8)

**Pull request merged**